lib/dom.js

```diff
@@ -88,6 +88,28 @@ declare class Event {
     BUBBLING_PHASE: number;
 }
 
+declare class UIEvent extends Event {
+    detail: number;
+}
+
+declare class MouseEvent extends UIEvent {
+    altKey: boolean;
+    button: number;
+    buttons: number;
+    clientX: number;
+    clientY: number;
+    ctrlKey: boolean;
+    metaKey: boolean;
+    movementX: number;
+    movementY: number;
+    region: number;
```

Comment:
Looks like this can be `null`:

> Returns the id of the hit region affected by the event. If no hit region is affected, null is returned.

`region: ?number;`